modules/rosa-create-an-identity-based-policy.adoc

```diff
@@ -18,7 +18,19 @@ You can create an identity-based Identity and Access Management (IAM) policy tha
 . In the IAM console, select *Policies* from the left navigation menu.
 . Click *Create policy*.
 . Select the *JSON* tab to define the policy using JSON format.
-. Copy and paste the following JSON policy document into the editor:
+. To get the IP addresses that you need to enter into the JSON policy document, run the following command:
++
+[source,terminal]
+----
+$ ocm get /api/clusters_mgmt/v1/trusted_ip_addresses
+----
++
+[NOTE]
+====
+These IP addresses are not permanent and are subject to change. You must continuously review the API output and make the necessary updates in the JSON policy document.
+====
++
+. Copy the output IP addresses and add all the IP addresses into the `"aws:SourceIp": []` array in your `policy_document.json`:
```

Comment:
From my perspective, these two steps (7 and 8) are confusing because they both refer to editing the policy document. Are you copying them into the editor in step 7, or are you editing this somewhere else and then copying it into the editor in the next step? I'm also not sure where the editor is.

If you can just edit the document directly in one place, I would remove one of the "Copy..." steps in favor of adding a step that orients the user to the editor.

Reply:
Hello @ousleyp I have updated two steps( 7 and 8) to avoid confusion.  Could you please take a look? Thank you!

Cc @wanghaoran1988 @yuwang-RH @nalhadef